apps/web/playwright/apple-calendar.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from "@playwright/test";
+
+import { test } from "./lib/fixtures";
+import { installAppleCalendar } from "./lib/testUtils";
+
+test.describe.configure({ mode: "parallel" });
+
+test.afterEach(({ users }) => users.deleteAll());
+
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+const APPLE_CALENDAR_EMAIL = process.env.E2E_TEST_APPLE_CALENDAR_EMAIL;
+// eslint-disable-next-line turbo/no-undeclared-env-vars
+const APPLE_CALENDAR_PASSWORD = process.env.E2E_TEST_APPLE_CALENDAR_PASSWORD;
+
+test.describe("Apple Calendar", () => {
+  if (!APPLE_CALENDAR_EMAIL || !APPLE_CALENDAR_PASSWORD)
+    throw new Error("It should run only it has the testing credentials");
```

Comment:
I'm aware that we usually do `test.skip` for checking variable values (as in [integrations-stripe.e2e.ts](https://github.com/calcom/cal.com/blob/6c7d7be9cc96a9d3359564e350117ca25e0f82e2/apps/web/playwright/integrations-stripe.e2e.ts#L20)), but in this case, I throw an error to have `APPLE_CALENDAR_EMAIL` and `APPLE_CALENDAR_PASSWORD` as strings, instead of manually do an `as string` when filling the inputs (don't handle types at all would cause a transpilation error on typescript because the `fill` method expects a string and can't be an undefined value)

Reply:
I noticed that this is NOT what should happen at all, I'll change it to `skip` to avoid breaking tests on pipelines that are not set up yet.

But not sure about how I should handle the typescript warning about these variables being undefined; can I just `const APPLE_CALENDAR_EMAIL = value as string`?

Reply:
You can null assert them like this: APPLE_CALENDAR_EMAIL!

Reply:
This `APPLE_CALENDAR_EMAIL!` also checks for undefined @zomars? Asking because the value it'll come as `undefined` probably

Reply:
We don't need to check for undefined here. That's what the skip is for. So if they're set we can assume they're not null.